Dockerfile

```diff
@@ -1,12 +1,19 @@
+FROM alpine:3.18
```

Comment:
This isn't really a change from the original.
`ARG RESTY_IMAGE_BASE="alpine"`
`ARG RESTY_IMAGE_TAG="3.18"`
`FROM ${RESTY_IMAGE_BASE}:${RESTY_IMAGE_TAG}`